pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -110,7 +122,14 @@ func TestCephObjectStoreUserController(t *testing.T) {
 			Namespace: namespace,
 		},
 		Spec: cephv1.ObjectStoreUserSpec{
-			Store: store,
+			Store:      store,
+			MaxBuckets: maxbucket,
+			MaxSize:    maxsize,
+			MaxObjects: maxobject,
+			Caps: &cephv1.CapSpec{
+				User:   "read",
+				Bucket: "read",
+			},
```

Comment:
You're adding things to the test config, but you aren't actually validating any of this. Tests need to validate that these settings are set. I expect to see tests for when only one type of quota is set (maxSize/maxObjects) and validating the default (-1) is set for the other. This also means that I expect to see tests being added and not merely existing tests being modified. You will need at **least** 6 new tests: quotas unset, one quota set and not another, all quotas set, maxBuckets set (verified unset in another test), caps set (verified unset in another test).

Reply:
@BlaineEXE added test cases